lib/fastlane/plugin/firebase_app_distribution/actions/firebase_app_distribution_action.rb

```diff
@@ -16,8 +16,9 @@ class FirebaseAppDistributionAction < Action
       FIREBASECMD_ACTION = "appdistribution:distribute".freeze
       BASE_URL = "https://firebaseappdistribution.googleapis.com"
       PATH = "/v1alpha/apps/"
+      TOKEN_PATH = "https://oauth2.googleapis.com/token"
```

Comment:
What kind of token?  `AUTH_TOKEN`?  Might consider giving a little more context in the name.

Reply:
Or maybe even `TOKEN_CREDENTIAL_URI` then it's easy to grep over to the way it gets used with the Oauth client